oak_runtime/src/node/http/mod.rs

```diff
@@ -143,9 +143,17 @@ impl HttpServerNode {
         });
 
         // Low-level server creation is needed, to be able to validate TLS streams.
-        let mut tcp = TcpListener::bind(&self.address)
-            .await
-            .expect("Could not create TCP listener.");
+        let mut tcp = match TcpListener::bind(&self.address).await {
+            Ok(tcp) => tcp,
+            Err(e) => {
+                error!(
+                    "{:?}: Could not create TCP listener: {:?}",
+                    std::thread::current().id(),
+                    e
+                );
+                return;
```

Comment:
So what happens in this case? Does the creator of the node get any details about the failure? Or does it just get logged here, and the creator will send messages to a black hole?

Reply:
I don't think it's much better &ndash; the pseudo-Node now terminates cleanly rather than `panic`king, which means the Runtime will clean up all of its channels, so attempting to wait-for/read an incoming HTTP invocation should fail (modulo #763).